tests/base.py

```diff
@@ -391,7 +391,7 @@ def records_data_type_conversions(self, records):
             ]
             object_keys = [
                 'discount', 'plan', 'coupon', 'status_transitions', 'period', 'sources', 'source',
-                'package_dimensions',
+                'package_dimensions','price'
```

Comment:
Why did we add 'price' in this variable?

Reply:
The price variable had a `created` field which had the epoch fromat. This function is to convert the epoch time to datetime for the fields mentioned in the list `timestamp_to_datetime_keys` which are present as nested fields in the `object_keys`.
Also added a code comment for the same.